demos-europe/demosplan-core
# fix(refs DPLAN-2834): adjust docx export hint

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-2834/Falscher-Hinweis-Daten-im-Export


Since the settings affect docx exports in landscape format as well in portrait format, the translations needs to be adjusted accordingly.

### How to review/test

- checkout
- login as FPA (e.g.)
- select procedure
- navigate to settings
- make sure the export hint for docx files has been adjusted

### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
